modules/ROOT/pages/dataweave-memory-management.adoc

```diff
@@ -4,20 +4,20 @@ include::_attributes.adoc[]
 endif::[]
```

Comment:
ORG
= DataWeave Memory Management
== RAM vs Disk Usage

Because there is only one subtopic, do you suppose we should reword the container title so that it encompasses that solitary subt, and omit the division? "Fine-Tune Large File Memory Management"? "Manage DataWeave RAM and Disk Storage"?

Or, we could simply omit the subtitle and include RAM vs. disk use in the topic intro text:

When processing large files through DataWeave in Mule runtime engine, consider the availability of RAM versus disk storage so that you can fine-tune how much memory is used, and when.